### PR TITLE
Fix a few scope related deprecation warnings

### PR DESCRIPTION
**Pull request by s-ludwig**

### CODE REVIEW

File: http/vibe/http/common.d [hunk 1]
```diff
@@ -300,7 +300,7 @@ class HTTPResponse {
 		InetHeaderMap headers;
 
 		/// All cookies that shall be set on the client for this request
-		@property ref DictionaryList!Cookie cookies() { return m_cookies; }
+		@property scope ref DictionaryList!Cookie cookies() return { return m_cookies; }
```
Comment (Geod24) [inline http/vibe/http/common.d:303]:
```suggestion
		@property ref DictionaryList!Cookie cookies() scope return { return m_cookies; }
```
?

File: stream/vibe/stream/multicast.d [hunk 2]
```diff
@@ -75,10 +75,10 @@ struct MulticastStream(OutputStreams...) {
 		if (m_tasks.length > 0) {
 			Exception ex;
 			foreach (i, T; OutputStreams[1 .. $])
-				m_tasks[i] = runTask({
-					try m_outputs[i+1].flush();
+				m_tasks[i] = runTask((MulticastStream _this) {
```
Comment (Geod24) [inline stream/vibe/stream/multicast.d:78]:
```suggestion
				m_tasks[i] = runTask((scope MulticastStream _this) {
```
?

**Review by Geod24 — APPROVED** [review]

File: http/vibe/http/common.d (hunk 1 above)
Comment (s-ludwig) [reply http/vibe/http/common.d:303]: Do we even need the `scope` at all here? I still couldn't force myself to look through the spec to fully grasp the `scope` additions (assuming that everything is correctly documented by now) and from the observed effects I'm unsure whether `return` a) implies `scope`, b) is required to be used in conjunction with `scope`, or c) has an independent effect on the parameter/`this` pointer.

File: stream/vibe/stream/multicast.d (hunk 2 above)
Comment (s-ludwig) [reply stream/vibe/stream/multicast.d]: Applied to both similar places.

File: http/vibe/http/common.d (hunk 1 above)
Comment (Geod24) [reply http/vibe/http/common.d]: We need `scope` only if we intend to call this function with a `scope DictionaryList!Cookie` is my understanding. But I have a similar issue with the spec.

**Pull request merged**